ansible/roles/ansible-tvault-contego-extension/templates/contego_service.j2

```diff
@@ -3,6 +3,7 @@ Description=Tvault contego
 [Service]
 User={{TVAULT_CONTEGO_EXT_USER}}
 Group={{TVAULT_CONTEGO_EXT_USER}}
+LimitNOFILE=65536
```

Comment:
Please avoid hardcoding in j2 templates. This can be moved to vars file and same variable can be referred in both j2 templates.

Reply:
Added New Variable contego_service_ulimits_count